gtk4/src/subclass/widget.rs

```diff
@@ -1341,6 +1341,20 @@ pub trait CompositeTemplate: WidgetImpl {
     fn bind_template(klass: &mut Self::Class);
 }
 
+pub trait CompositeTemplateClass {
```

Comment:
This needs some docs, I guess. Also other things in here

Reply:
Added some docs, they should probably be in the subclass prelude as they are useless outside of subclasses